tests/phpunit/elementor/modules/image-loading-optimization/test-module.php

```diff
@@ -0,0 +1,68 @@
+<?php
+namespace Elementor\Tests\Phpunit\Elementor\Modules\Image_Loading_Optimization;
+
+use Elementor\Core\Base\Document;
+use Elementor\Modules\PageTemplates\Module as PageTemplatesModule;
+use Elementor\Plugin;
+use ElementorEditorTesting\Elementor_Test_Base;
+use Elementor\TemplateLibrary\Source_Local;
```

Comment:
A few of these imported classes don't seem to be used. Can we remove them? Alternately, it looks like the places where we are using the modules manager to get the `page-templates` module, we could instead use the `PageTemplatesModule` class that is being imported here and avoid some undefined method and undefined class constant warnings.

```suggestion
use Elementor\Plugin;
use ElementorEditorTesting\Elementor_Test_Base;
```

Reply:
Fixed.